addons/account/models/account_move.py

```diff
@@ -2117,10 +2117,15 @@ def compute_tax_repartition_lines_mapping(move_vals):
                         base_line = self.line_ids.filtered(lambda line: tax in line.tax_ids.flatten_taxes_hierarchy())[0]
                         account_id = base_line.account_id.id
 
+                tags = refund_repartition_line.tag_ids
+                if line_vals.get('tax_ids'):
+                    subsequent_taxes = self.env['account.tax'].browse(line_vals['tax_ids'][0][2])
```

Comment:
I suppose we can assume that it will always be a command 6 since it is already used a few lines below

Reply:
indeed